docs/engineering-principles.md

```diff
@@ -0,0 +1,48 @@
+# Engineering Principles
+Engineering principles define the core tenets and beliefs that the engineering team and all contributors are expected to keep front of mind. These tenets are meant to reflect the values of the organization as well as the philosophy upon which the software was founded. Principles typically come in the form of punchy one line statements, but a great deal of depth can be written on each one to further inform of the intent of the statement.
```

Comment:
Since these are aspirational goals rather than a reflection of current or past practices of our team, it seemed better to omit the part where it suggests the software is founded on these principles. It's misleading, and the sentence seems fine without that part.
```suggestion
Engineering principles define the core tenets and beliefs that the engineering team and all contributors are expected to keep front of mind. These tenets are meant to reflect the values of the organization. Principles typically come in the form of punchy one line statements, but a great deal of depth can be written on each one to further inform of the intent of the statement.
```